src/exceptions.md

```diff
@@ -216,7 +217,7 @@ Contents of section .vector_table:
 
```

Comment:
We should be more elaborate here and explain what the reader is looking at.

1. What is the interesting part of the .text disassembly? @japaric what did you have in mind?
2. I'm not a Cortex-M user, so I need some help with the reset table.
  1. What is the first entry for? `LONG(ORIGIN(RAM) + LENGTH(RAM));`
  2. We can explain that the reader can see first entry, reset vector address, and then the pattern of vectors that is assigned above (5 default - 4 zero - 1 default - 2 zero - 2 default).
  3. Why is the address in the vector table `79`, but in the disassembly it shows address `78` for the default handler?

Reply:
Yes, definitively.

We should link back to https://rust-embedded.github.io/embedonomicon/memory-layout.html#inspecting-it because we have already covered the first two entries of the vector table there (re: i). We also briefly mentioned why the low bit of the address is set to 1 (thumb mode, re: iii).

re: ii, we should have the user compare the contents of the vector table to [the table depicted in the ARM docs](https://developer.arm.com/docs/dui0552/latest/the-cortex-m3-processor/exception-model/vector-table), and verify that the reserved slots are indeed zero and that all other vectors point to the default handler (0x78 + 1).

> What is the interesting part of the .text disassembly? @japaric what did you have in mind?

Mainly the answer to ii. See previous paragraph.